c7n/resources/vpc.py

```diff
@@ -2185,6 +2185,78 @@ class resource_type(query.TypeInfo):
         taggable = False
 
 
+@KeyPair.filter_registry.register('unused')
+class UnusedKeyPairs(Filter):
+    """Filter for used or unused keys. The default is unused
```

Comment:
general styling on doc strings. one sentence summary ideally one line. followed by paragraph space for longer description.

ie. 

```
"""Filter for used or unused keys.

The default...
"""
```

Reply:
improved in f93f217